apps/graduate/src/widgets/sidebar/model/model.ts

```diff
@@ -0,0 +1,28 @@
+export type MenuItem = {
+  label: string
+  to: string
+}
+
+export type MenuSection = {
+  title: string
+  items: MenuItem[]
+}
+
+export const adminMenuSections: MenuSection[] = [
+  {
+    title: '메뉴 관리',
+    items: [
+      { label: '공지사항', to: '/notices' },
+      { label: '안내 및 내규', to: '/guidelines-and-rules' },
+      { label: '진행 일정', to: '/schedule' },
+    ],
+  },
+  {
+    title: '졸업생 관리',
+    items: [
+      { label: '대상자 전체 관리', to: '/graduates-all' },
+      { label: '자격증 신청 관리', to: '/certification-management' },
+      { label: '논문 신청 관리', to: '/thesis-application-management' },
+    ],
+  },
+]
```

Comment:
다른 곳에서 사용되지 않는 타입이라면 따로 타입을 선언하지 않고 `as const` 사용해서 const assertion 사용해 주시면 좋을 것 같아요.
```suggestion
export const adminMenuSections = [
  {
    title: '메뉴 관리',
    items: [
      { label: '공지사항', to: '/notices' },
      { label: '안내 및 내규', to: '/guidelines-and-rules' },
      { label: '진행 일정', to: '/schedule' },
    ],
  },
  {
    title: '졸업생 관리',
    items: [
      { label: '대상자 전체 관리', to: '/graduates-all' },
      { label: '자격증 신청 관리', to: '/certification-management' },
      { label: '논문 신청 관리', to: '/thesis-application-management' },
    ],
  },
] as const
```